src/components/EventList.tsx

```diff
@@ -60,15 +65,27 @@ export default function EventList ({ eventResponses, events, onSignup, onUndoSig
 }
 
 interface EventListItemProps {
+    booked: BookedEvent | undefined;
```

Comment:
Is it necessary for this to be of type `BookedEvent`? I think it would be cleaner to just use a boolean.